### ISSUE TITLE
The docs have an issue on intro.md

### ISSUE THREAD

**Issue by wsmigielski** [Kraigie/nostrum#10]
```elixir
def deps do
  [{:nostrum, "~> 1.0"]
end
```
should be
```elixir
def deps do
  [{:nostrum, "~> 0.1"]
end
```

**Comment by cronokirby** [comment]
it can even be bumped up to `{:nostrum, "~> 0.2.1"}`

**Comment by Kraigie** [comment]
It could be, but I don't want to have to update it in the event I start making a lot of breaking changes and bug fixes up to a 1.0 release. 

Fixed dev docs in 281215a820a8b8559e5331914482557339b38eae, thanks. Official docs will have to wait for a release.
